### ISSUE TITLE
Connecting Remote Storage

### ISSUE THREAD

**Issue by j0nr** [Laverna/laverna#588]
I have just set up my own remote storage server (which I believe is functional - although not sure how to verify. I can log into it. I am using php-remote-storage) 

I have locally installed Laverna. When I go to the (functioning) Laverna on my server, enter user and press the "connect remote storage" button (bottom right) it just kinda does nothing. It thinks for split second then reverts to "connect remote storage". 

How do I begin troubleshooting what's not working here please?

I am running both php-remote-storage and laverna on the same, Ubuntu server.

Thanks.

Jon

**Comment by j0nr** [comment]
Here is a shot of the console after I pressed the connect remote storage button.

![lav_concole](https://cloud.githubusercontent.com/assets/2248120/17779340/4014447e-655f-11e6-900b-5097430d839e.PNG)

**Comment by wwebfor** [comment]
I think you're trying to connect to a RemoteStorage server from a different domain. Is it the case?

**Comment by j0nr** [comment]
Hi, 

The remote storage is on a subdomain. Like:

Laverna : example.com/notes
Remote storage: https://storage.example.com

**Comment by j0nr** [comment]
OK I added this to remote storage apache server conf:

    Header set Access-Control-Allow-Origin "*"

And now it works

**Comment by j0nr** [comment]
Actually...today it appears not to be working <confused>

It works on my Android Chrome phone...but not on Desktop Chrome or Firefox. I don't know what to say (or do).

**Comment by j0nr** [comment]
Right. I now have it working in all browsers, remote storage says its connected but I am not getting notes synced between devices.

**Comment by danieldaeschle** [comment]
same
